### PR TITLE
Homepage polish

### PR DESCRIPTION
**Pull request by austinmargarone**
@brandonetter Here's my final PR before I move on to dark mode styling. 

I'm having the following issues that are blocking me:
Head shot in the about section is very small
Breakpoints and margins are better but some parts of the page are cut off on medium devices
Contact group image on mobile looks different than the design
Contact group image on desktop has a weird space a the top
And whatever else issues that you notice.

### CODE REVIEW

File: components/homepage/Contact.tsx [hunk 1]
```diff
@@ -22,10 +27,15 @@ const Contact = () => {
...
         <div className="contact relative flex h-[330px] w-[1270px] shrink-0">
           <div className="absolute">
-            <Image src="/Group1.svg" width={1270} height={330} alt="Ellipse" />
+            <Image
+              src="/Group1.svg"
+              width={1270}
+              height={330}
+              alt="Background Desktop"
+            />
           </div>
```
Comment (brandonetter) [inline components/homepage/Contact.tsx]: try instead:
```js
{/* Desktop */}
<div className="mx-auto hidden w-full md:block md:max-w-[80rem]">
        <div className="contact relative flex h-[330px] shrink-0">
          <div className="absolute h-full w-full">
            <Image
              src="/Group1.svg"
              className="object-cover"
              fill={true}
              alt="Background Desktop"
            />
          </div>
```

Comment (brandonetter) [reply components/homepage/Contact.tsx]: You can then rework your button and text to not use margins and instead use justify between.

The way you had your button positioned by having a huge margin the left meant you couldn't shrink the elements as the page shrunk- or your text would get cut-off. I think this is a better approach.

```js
 <div className="absolute flex w-full justify-between px-[4rem] ">
            <h2 className=" mt-[5.19rem] w-[32.5rem] text-left">
              Have a project in mind that requires technical expertise?
            </h2>
            <div className=" mt-[10.69rem] flex">
              <button className="contactbtnlg flex">
                <p className="boldbody text-white">Get in Touch with Me</p>
                <Image src="/Arrow2.svg" width={16} height={0} alt="Ellipse" />
              </button>
            </div>
 ```
 
 These changes should give you something closer to what you're looking for
 

https://github.com/austinmargarone/portfolio/assets/4108484/0f7ed4b8-5eac-4dee-956b-16fd40011442

**Review by brandonetter — CHANGES_REQUESTED** [review]
Some notes:

image

We may want to center this is a bit better. It's aligned to the left.


Medium devices have a lot of room to play with:
image

Cutoff:
image

The issue is your width is being hard-coded so it doesn't shrink. Instead, try making the width something relative like 'full' in the parent. And if we want to contain the Image a little better try setting it's `fill` state to true and giving it's parent some dimensions. I've noted the small changes to get you on the right track.

**Comment by austinmargarone** [comment]
@brandonetter How about the head shot image in the about section? and the group image from the mobile contact card, when I group the images it distorts it.

**Pull request merged**